<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_interface</class>
 <widget class="QWidget" name="user_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>user_interface</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLineEdit" name="priorityLineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="durationButton">
         <property name="text">
          <string>Show Duration</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="durationLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
